bin/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+dotnet tool restore && dotnet build -o exe
```

Comment:
I like these shortcuts for doing stuff. Though I don't see the documentation here for them, I'm guessing they are added down the road but leaving a comment for my own sake.